index.php

```diff
@@ -0,0 +1,31 @@
+<?php
+header("Cache-Control: no-store, no-cache, must-revalidate");
```

Comment:
Stumbled across this timely blog post https://www.fastly.com/blog/headers-we-dont-want

Apparently `Pragma` has no meaning in responses. According to the post `Cache-Control: no-cache, private` should be all you need to make sure the page isn't cached, but these extra headers likely shouldn't do harm.